notes/links.txt

```diff
@@ -0,0 +1,300 @@
+"station1","station2","line"
```

Comment:
this file is just a bunch of dumb notes I made when mapping things out. Keeping for the moment to refer back to.